Intersect (Core)/GameObjects/LightBase.cs

```diff
@@ -68,7 +68,7 @@ Color color
         /// <returns></returns>
         public override int GetHashCode()
         {
-            return Intensity ^ (int)Expand ^ (Color?.GetHashCode() ?? 0);
+            return Intensity ^ (int)Expand ^ (Color?.GetHashCode() ?? 0) ^ TileX.GetHashCode() ^ TileY.GetHashCode();
```

Comment:
Change to `HashCode.Combine(...)`, don't pass `GetHashCode()` into the `Combine(...)` method

Reply:
return HashCode.Combine(Intensity, (int)Expand, Color, TileX, TileY);

Will work for you Panda?

Reply:
@TimtyG Don't cast to int here, just pass Expand directly into the method

Reply:
Alright, on it